geometry_msgs/Point start
geometry_msgs/Point stop
---
float64 cost
